.github/workflows/Tests.yml

```diff
@@ -86,15 +86,24 @@ jobs:
           cd zimui
           $(yarn bin)/cypress run
 
-  build-docker:
+  # this job replaces the standard "build_docker" job since it builds the docker image
+  run-integration-tests:
     runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v3
 
       - name: Ensure we can build the Docker image
         run: |
-          docker build -t testimage .
+          docker build -t youtube2zim .
 
       - name: Ensure we can start the Docker image
```

Comment:
Do you see any significant advantage in keeping this step isolated? This was meant to be a minimal integration test (can we at least start the container and the process within) but now that we have more thorough integration tests, I'm not sure there is any benefit to keeping this step

Reply:
Ah I see. It is redundant. I'll remove it.

Reply:
Removed in [bb30caf](https://github.com/openzim/youtube/pull/258/commits/bb30caf8be6113c27ff74c80aa2f2ec99c5cdde9)